recipes/cpython/all/conandata.yml

```diff
@@ -1,65 +1,79 @@
 sources:
-  "3.10.0":
-    url: "https://www.python.org/ftp/python/3.10.0/Python-3.10.0.tgz"
-    sha256: "c4e0cbad57c90690cb813fb4663ef670b4d0f587d8171e2c42bd4c9245bd2758"
-  "3.9.7":
-    url: "https://www.python.org/ftp/python/3.9.7/Python-3.9.7.tgz"
-    sha256: "a838d3f9360d157040142b715db34f0218e535333696a5569dc6f854604eb9d1"
-  "3.8.12":
-    url: "https://www.python.org/ftp/python/3.8.12/Python-3.8.12.tgz"
-    sha256: "316aa33f3b7707d041e73f246efedb297a70898c4b91f127f66dc8d80c596f1a"
+  "3.12.2":
+    url: "https://www.python.org/ftp/python/3.12.2/Python-3.12.2.tgz"
+    sha256: "a7c4f6a9dc423d8c328003254ab0c9338b83037bd787d680826a5bf84308116e"
+  "3.11.9":
+    url: "https://www.python.org/ftp/python/3.11.9/Python-3.11.9.tgz"
+    sha256: "e7de3240a8bc2b1e1ba5c81bf943f06861ff494b69fda990ce2722a504c6153d"
+  "3.10.14":
+    url: "https://www.python.org/ftp/python/3.10.14/Python-3.10.14.tgz"
+    sha256: "cefea32d3be89c02436711c95a45c7f8e880105514b78680c14fe76f5709a0f6"
+  "3.9.19":
+    url: "https://www.python.org/ftp/python/3.9.19/Python-3.9.19.tgz"
+    sha256: "f5f9ec8088abca9e399c3b62fd8ef31dbd2e1472c0ccb35070d4d136821aaf71"
+  "3.8.19":
+    url: "https://www.python.org/ftp/python/3.8.19/Python-3.8.19.tgz"
+    sha256: "c7fa55a36e5c7a19ec37d8f90f60a2197548908c9ac8b31e7c0dbffdd470eeac"
 patches:
-  "3.10.0":
-    - patch_file: "patches/3.10/3.10.0-0001-msvc.patch"
-      patch_description: "Version specific patches to MSVC projects to allow injection of dependencies"
+  "3.12.2":
+    - patch_file: "patches/3.9/3.9.7-0002-_msi-vcxproj.patch"
+      patch_description: "Fix ARM/ARM64 mismatch in project file"
+      patch_type: "bugfix"
+    - patch_file: "patches/3.12/3.12.1-0001-_ctypes-ffi.patch"
+      patch_description: "Support shared libffi"
+      patch_type: "portability"
+    - patch_file: "patches/3.x-0001-relocatable-python-config.patch"
+      patch_description: "Allow package to be relocatable"
+      patch_type: "conan"
+    - patch_file: "patches/3.12/3.12.1-0002-remove-module-deps.patch"
+      patch_description: "Remove section of solution file forcing projects to be built that might not be used for this recipe"
+      patch_type: "conan"
+  "3.11.8":
```

Comment:
@Ahajha is this a typo?

Reply:
Ah, yea it is. I have another PR out, I'll fix it in that one.

Reply:
Thanks, great!